superset/migrations/versions/2a6878fdc2bd_.py

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""empty message
+
+Revision ID: 2a6878fdc2bd
+Revises: ('134cea61c5e7', '19e978e1b9c3')
+Create Date: 2021-04-09 08:03:21.449796
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "2a6878fdc2bd"
+down_revision = ("134cea61c5e7", "19e978e1b9c3")
```

Comment:
I think you can fix this by changing the `down_revision` on your previous migration rather than adding a new migration, which would be a bit cleaner.